CONTRIBUTING.md

```diff
@@ -0,0 +1,32 @@
+# Contributing
+
+## Adding flags
+
+Flags are stored in the `flags` folder. To add a new one simply:
+
+1. Create a new js file with the id of the flag `flagid.js`
+2. Fill the flag out with it's info.
```

Comment:
`it's info` is a bit vague on its own, so it could be nice to add a reference like:

```suggestion
2. Fill the file out with the flag's info. You can use the format of [`pride_1978_sevenStripes.js`](./data/flags/pride_1978_sevenStripes.js) as a reference.
```